src/components/tooltip/tooltip.tsx

```diff
@@ -41,8 +43,10 @@ import { createRandomString } from '../../util/random-string';
  * effortlessly recognize can be hovered.
  *
  * @exampleComponent limel-example-tooltip
+ * @exampleComponent limel-example-tooltip-max-character
  * @private
```

Comment:
Hmm… @jgroth, do you know if there's any particular reason this is still private? Is it because we weren't sure about the structure of the component?

Reply:
I cannot remember. I think we just forgot to remove it :slightly_smiling_face: